dotnet-developer-projects.md

```diff
@@ -342,6 +342,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [Material Design in XAML Toolkit](https://github.com/ButchersBoy/MaterialDesignInXamlToolkit) - WPF/XAML resources, styles & templates for creating applications using Material Design
   * [Mono XWT](https://github.com/mono/xwt) - A cross-platform UI toolkit for creating desktop apps
   * [UWP Community Toolkit](https://github.com/Microsoft/UWPCommunityToolkit) - The UWP Community Toolkit is a collection of helper functions, custom controls, and app services. It simplifies and demonstrates common developer tasks building UWP apps for Windows 10.
+  * [Neutronium](https://github.com/NeutroniumCore/Neutronium) - Neutronium is a library to create .NET desktop applications using HTML, CSS and javascript. It uses MVVM pattern exactly the same way as WPF application.
```

Comment:
MVVM pattern --> the MVVM pattern
application --> applications.

Reply:
@rpetrusha , Thanks for the feedback. I just commited the changes.